src/Microsoft.OData.Client/ALinq/ExpressionWriter.cs

```diff
@@ -508,6 +525,23 @@ internal override Expression VisitConstant(ConstantExpression c)
                 ODataEnumValue enumValue = new ODataEnumValue(memberValue, typeNameInEdm ?? typeAnnotation.ElementTypeName);
                 result = ODataUriUtils.ConvertToUriLiteral(enumValue, CommonUtil.ConvertToODataVersion(this.uriVersion), null);
             }
+            else if (m != null && ReflectionUtil.IsSequenceMethod(m.Method, SequenceMethod.Contains))
+            {
+                var csvBuilder = new StringBuilder();

```

Comment:
>var [](start = 16, length = 3)

In this library, we generally prefer to use actual types instead of "var" as it tends to increase readability and validate expectations.  Could you please replace the occurances of "var" with the actual type?